TeknoParrotUi.Common/GameProfiles/ArcadeLove.xml

```diff
@@ -2,13 +2,13 @@
 <GameProfile xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
   <GameName>Ge-Sen Love: Plus Pengo!</GameName>
   <GamePath/>
-  <TestMenuParameter>-t</TestMenuParameter>
```

Comment:
how do you access test menu then? if its ingame, change also <HasSeparateTestMode> to false

Reply:
Ingame.